File: App/app/src/main/java/com/example/apiverte_colony_management/DTOs/TypicalInspectionDTO.java
package com.example.apiverte_colony_management.DTOs;

import java.time.LocalDateTime;
import java.util.UUID;

public class TypicalInspectionDTO {
    public UUID Id;

    public String Name;

    public Boolean IsActive;

    public String CreatedBy;

    public long CreatedDate;
    public String LastModifiedBy;
    public long LastModifiedDate;

    public UUID UserId;

    public UUID ColonyId;

    public String[] Weather;
    public String Population;
    public String Mood;
    public String Fitness;

    public int BroodChambers;
    public int HoneyChamber;
    public Boolean MouseGuard;
    public Boolean WaspGuard;
    public Boolean PollenCollector;
    public String HiveBottom;
    public String Vents;

    public String Brood;
    public String Honey;
    public String BroodPattern;
    public String[] Issues;

    public String[] Growth;
    public String[] Seasonal;

    public String Status;
    public String Cells;
    public String SwarmStatus;
    public Boolean Excluder;
}

File: Server/CQS/DTOs/TypicalInspectionDto.cs
using System;

namespace Server.CQS.DTOs
{
    public class TypicalInspectionDto
    {
        public Guid Id { get; set; }
        public string CreatedBy { get; set; }
        public long CreatedDate { get; set; }
        public string LastModifiedBy { get; set; }
        public long LastModifiedDate { get; set; }
        public bool IsActive { get; set; }
        public Guid? UserId { get; set; }

        public Guid ColonyId { get; set; }

        public string[] Weather { get; set; }
        public string Population { get; set; }
        public string Mood { get; set; }
        public string Fitness { get; set; }

        public int BroodChambers { get; set; }
        public int HoneyChamber { get; set; }
        public bool MouseGuard { get; set; }
        public bool WaspGuard { get; set; }
        public bool PollenCollector { get; set; }
        public string HiveBottom { get; set; }
        public string Vents { get; set; }

        public string Brood { get; set; }
        public string Honey { get; set; }
        public string BroodPattern { get; set; }
        public string[] Issues { get; set; }

        public string[] Growth { get; set; }
        public string[] Seasonal { get; set; }

        public string Status { get; set; }
        public string Cells { get; set; }
        public string SwarmStatus { get; set; }
        public bool Excluder { get; set; }
    }
}
